ui/packages/catalog-ui-search-e2e/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "catalog-ui-search-e2e",
+  "version": "0.0.1",
```

Comment:
does the yarn.lock file change as a result of this?

Reply:
This should definitely be verified.

Reply:
I don't think it matters

Reply:
no change